recipes/ca-certificates/bld.bat

```diff
@@ -0,0 +1,16 @@
+:: Windows really uses a different store of certificates (see
+::   http://superuser.com/questions/411909/where-is-the-certificate-folder-in-windows-7)
+
+:: The certs here are probably only useful for unix-type apps, such as git, that don't necessarily
+::    respect the windows way of doing things.
+
+:: version at time of writing is 1.25
+curl -O https://raw.githubusercontent.com/curl/curl/master/lib/mk-ca-bundle.pl
+if ( $($(CertUtil -hashfile mk-ca-bundle.pl SHA256)[1] -replace " ","") -eq "6fbdd1c76a41b7ab41cd616b19e52522e2b7efb636120950f053ef2c22de44af" ) { echo "mk-ca-bundle download ok" } else {echo "mk-ca-bundle.pl checksum bad.  Has it changed?" && exit 1}
```

Comment:
Hasn't changed. I am comparing the exact same sha256 in the Mac and Linux case. It is working down there. Not sure why this one would be different. :/